crates/apub/src/objects/comment.rs

```diff
@@ -141,6 +141,15 @@ impl Object for ApubComment {
     check_apub_id_valid_with_strictness(note.id.inner(), community.local, context).await?;
     verify_is_remote_object(note.id.inner(), context.settings())?;
     verify_person_in_community(&note.attributed_to, &community, context).await?;
+
+    let old_comment = note.id.dereference_local(context).await;
+    let old_timestamp = old_comment
+      .as_ref()
+      .map(|c| c.updated.unwrap_or(c.published))
+      .clone()
+      .ok();
+    verify_object_timestamp(old_timestamp, note.updated.or(note.published))?;
```

Comment:
Unfortunately this is rather verbose, no way to avoid that unless we add a trait (which would be even more verbose and wouldnt be used anywhere else).